docs/THEMES.md

```diff
@@ -5,6 +5,10 @@ Themes are CSS files that act as themes by containing many different css variabl
 Most of these variables are small variations of the base colors of the theme so using scss allows us to generate them from the base colors to save us work.
 But of course we can overwrite those variables manually too, we can even overwrite css classes if our theme needs it.
 
+## Notes
+
+- The account sidebar should not be themed
```

Comment:
In "light" mode the sidebar becomes a bit disturbing with such a hard contrast, I don't think this needs to be discussed now, but are we really set on never changing this ever? 😄

Reply:
depends on what we want to do, my last state of discussion with @r10s was that we want to allow different themes for different accounts.
If we stay with that idea then it makes sense to theme the sidebar separately (maybe offer a dark vs light mode switch).

If we make it themable and don't support per-account themes then we could even do crazy gradient designs like discord in the future: https://discord.com/blog/bring-your-vibe-to-discord-with-new-themes-in-nitro

Reply:
I like that!